bin/misc/meson.build

```diff
@@ -22,7 +26,7 @@ executable(
     ['fce.c'],
     include_directories: root_includes,
     link_with: libatalk,
-    dependencies: [iniparser],
+    dependencies: [iniparser,iconv_deps],
```

Comment:
nitpick: the common coding style in this project is that a comma is always followed by a space